### PR TITLE
Allow Custom Button Name with Collapsible Component

### PR DESCRIPTION
**Pull request by jswongF5**
I was using the `Collapsible` component in a different spot that isn't a solution so removing the word **Solution**

### CODE REVIEW

**Comment by aknot242** [comment]
Why not default to "Show" and "Hide", then pass in alternate terms if you need to override them? This will make it more flexible.

**Comment by jswongF5** [comment]
Yup, you're sure right!

File: src/app/components/collapsible.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import React, { useState } from "react";
  * @param {React.ReactNode} prop.children  - The content to display of the container.
  * @returns {JSX.Element} The Collapsible component.
  */
-export function Collapsible({ title, children }) {
+export function Collapsible({ title, showName = 'Show', hideName = 'Hide', children }) {
```
Comment (aknot242) [inline src/app/components/collapsible.js:12]: Can you please move these optional params to the end of the function signature, then please update the jsdoc above it with descriptions of the new parameters?

**Review by aknot242 — CHANGES_REQUESTED** [review]

File: src/app/components/collapsible.js (hunk 1 above)
Comment (aknot242) [reply src/app/components/collapsible.js:12]: Oh, and please double-quote the strings while you are at it

Comment (jswongF5) [reply src/app/components/collapsible.js]: Still learning JS and/or NextJS.. sounds like `children` IS a valid variable that is assumed here.. Not for me to be confused by "also include everything else that might came along when someone uses this function, aka `...prop`).

**Pull request merged**